stable/docker-registry/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "docker-registry.fullname" . }}
+  labels:
+    app: {{ template "docker-registry.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "docker-registry.name" . }}
+      release: {{ .Release.Name }}
+{{ toYaml .Values.podDisruptionBudget | indent 2 }}
```

Comment:
I am pretty sure this will not do what you expect.
This will insert the contents of the variable without the name.

Reply:
It's supposed to do that
```
➜  ~/pr/c/charts/s/docker-registry git:(docker-registry-pdb) ✗ grep -A1 Budget values.yaml
podDisruptionBudget:
  maxUnavailable: 1
➜  ~/pr/c/charts/s/docker-registry git:(docker-registry-pdb) ✗ helm template -x templates/poddisruptionbudget.yaml --values ./values.yaml .
---
# Source: docker-registry/templates/poddisruptionbudget.yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: release-name-docker-registry
  labels:
    app: docker-registry
    chart: docker-registry-1.8.1
    release: release-name
    heritage: Tiller
spec:
  selector:
    matchLabels:
      app: docker-registry
      release: release-name
  maxUnavailable: 1
```
That looks good to me, see pdb [spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#poddisruptionbudgetspec-v1beta1-policy)